Makefile

```diff
@@ -100,10 +100,22 @@ else
 endif
 ifneq (, $(findstring adx, $(PROC)))
 	ADX = "y"
-	TFW_CFLAGS += -DADX=1
+else
+ifdef M
+	DIR = $(M)/
+endif
+	# Some cloud providers hide ADX support bit in vCPU, but it still present, make run-rime check to discard false negative cases
+	CHECK_CONF = $(DIR)scripts/check_conf.pl
+	ADX_SUPPORTED := $(shell $(CHECK_CONF) 2>/dev/null | grep ADX | if grep -q ': found'; then echo y; fi)
```

Comment:
Too wide lines, we keep lines in 80 characters long https://github.com/tempesta-tech/tempesta/blob/master/CodingStyle